pages/sign-in.tsx

```diff
@@ -40,7 +40,16 @@ function Dashboard() {
     }
     // Oddly this is needed for sign up from Google on Firefox. Don't know why, but
     // oauth params aren't grabbed from the URL automagically like on other browsers
-    void supabase.auth.getSession();
+    const getSession = async () => {
+      const session = await supabase.auth.getSession();
+      if (!session.data.session) return;
+      Object.entries(session.data.session).forEach(([key, value]) => {
+        if (value && typeof value === "string") {
+          localStorage.setItem(key, value);
+        }
+      });
+    };
+    getSession();
   }, [router]);
 
   // When they sign in, check if they have a join_id in localStorage and join the org if they do
```

Comment:
In `signInComponent.tsx`, we need to change this
![Screenshot from 2023-09-19 16-59-17](https://github.com/Superflows-AI/superflows/assets/33871096/79a2dcff-ad04-4811-8efd-b40b5593b69c)